ui/src/main/resources/CKEditor/Translations.xml

```diff
@@ -58,7 +58,7 @@ ckeditor.admin.reset.confirmation=Are you sure you want to restore the default c
 ckeditor.plugin.link.options=Options
 ckeditor.plugin.link.anchor=Anchor
 ckeditor.plugin.link.queryString=Query String
-ckeditor.plugin.link.defaultLabel=type the link label
+ckeditor.plugin.link.defaultLabel=Type the link label
```

Comment:
It's not for 2 different use cases, the key is used as value so it gets inserted inside the text.